### PR TITLE
add the task to open the documentation specific of the template and python3Console documentation

### PR DESCRIPTION
**Pull request by andreriesco**
Signed-off-by: Andre Riesco <andre.riesco@toradex.com>
Related-to: TIE-730

### CODE REVIEW

File: scripts/createFromTemplate.ps1 [hunk 1]
```diff
@@ -187,6 +187,11 @@ Get-ChildItem -Force -File -Recurse * | ForEach-Object {
                 ForEach-Object {
                     $_ -replace "__home__",$env:HOME
                 } | Set-Content $a
+                ( Get-Content $a ) |
+                ForEach-Object {
+                    # Get just the last string of the path, which is the name of the folder of the template
+                    $_ -replace "__templateFolder__",$(Split-Path -Path $templateFolder -Leaf)
```
Comment (microhobby) [inline scripts/createFromTemplate.ps1:193]: We have the `$template` variable that is the "name" of the template that can be used here

File: scripts/createFromTemplate.ps1 [hunk 2]
```diff
@@ -187,6 +187,11 @@ Get-ChildItem -Force -File -Recurse * | ForEach-Object {
                 ForEach-Object {
                     $_ -replace "__home__",$env:HOME
                 } | Set-Content $a
```
Comment (microhobby) [inline scripts/createFromTemplate.ps1:189]: Please add a break line here

**Review by microhobby — CHANGES_REQUESTED** [review]

File: scripts/createFromTemplate.ps1 [hunk 3]
```diff
@@ -187,6 +187,11 @@ Get-ChildItem -Force -File -Recurse * | ForEach-Object {
...
+                ( Get-Content $a ) |
+                ForEach-Object {
+                    # Get just the last string of the path, which is the name of the folder of the template
```
Comment (microhobby) [inline scripts/createFromTemplate.ps1:192]: Let's keep the code and comments where possible with 80 columns. The code style is not specified, I know, but I'll open an issue for that.

**Review by microhobby — CHANGES_REQUESTED** [review]

File: assets/tasks/common.json [hunk 4]
```diff
@@ -1017,6 +1017,35 @@
...
+        {
+            "label": "show-project-documentation",
+            "command": "${command:markdown.showPreview}",
+            "type": "process",
+            "dependsOn": [
+                "open-project-documentation-file"
+            ],
+            "dependsOrder": "sequence",
+            "problemMatcher": "$msCompile",
+            "icon": {
```
Comment (microhobby) [inline assets/tasks/common.json:1030]: add here:

```json
"presentation": {
                "echo": false,
                "reveal": "never",
                "focus": false,
                "panel": "shared",
                "showReuseMessage": false,
                "clear": true
            },
```

so the output panel is not presented and focused, catching the attention ...

**Review by microhobby — CHANGES_REQUESTED** [review]

File: assets/templatesDocumentation/python3Console/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ "Developer: Reload Window")
+
+To install a Python package on the venv (and also on the Debug image), put it 
+on the "requirements-debug.txt" file (to install it on the Release image put it
+on the "requirements.release.txt" file)
+
+For more details, please watch the demonstration video below:
+
+<video controls>
+  <source src="../common/python3/createEnvPython3.mp4" type="video/mp4">
```
Comment (microhobby) [inline assets/templatesDocumentation/python3Console/README.md:21]: This is returning `Error 401`, to solve this we will host the files on https://github.com/toradex/torizon-experimental-torizon-ide-v2-docs to not leave the repo here overloaded.

**Review by microhobby — CHANGES_REQUESTED** [review]

File: assets/tasks/common.json [hunk 6]
```diff
@@ -1017,6 +1017,35 @@
...
+        {
+            "label": "open-project-documentation-file",
+            "command": "code",
+            "detail": "hide",
+            "type": "process",
+            "args": [
+                "${userHome}/.apollox/assets/templatesDocumentation/__templateFolder__/README.md"
+            ],
+            "dependsOrder": "sequence",
+            "problemMatcher": "$msCompile",
```
Comment (microhobby) [inline assets/tasks/common.json:1044]: Also here, add the `presentation` definitions

**Review by microhobby — CHANGES_REQUESTED** [review]

**Comment by microhobby** [comment]
The video:

https://user-images.githubusercontent.com/2633321/212734759-54852563-cc33-4f63-90ad-2509907b647f.mp4

**Comment by andreriesco** [comment]
@microhobby I updated this MR with what we discussed, getting the README.md and all the rest of the media documentation (images, videos, etc) from the newly created [vscode-torizon-templates-documentation](https://github.com/toradex/vscode-torizon-templates-documentation) repo. Please review

**Comment by microhobby** [comment]
LGTM

**Review by microhobby — APPROVED** [review]

**Pull request merged**